runtime/src/prioritization_fee.rs

```diff
@@ -29,6 +41,22 @@ impl PrioritizationFeeMetrics {
         saturating_add_assign!(self.total_update_elapsed_us, val);
     }
 
+    fn update_prioritization_fee(&mut self, fee: u64) {
+        if fee == 0 {
+            saturating_add_assign!(self.non_prioritized_transactions_count, 1);
```

Comment:
iirc from our earlier conversation, I think we wanted to get the range of non-zero fees. If that's the case, we should return early here.

Reply:
OK, so we want to use non-fee/fee txs counters and fee range together. The fee range only applies to fee-paying txs. Updated comments and return early in df93cf58885b48b521e62c41bc1613a7673faed9